ocs_ci/ocs/platform_nodes.py

```diff
@@ -908,6 +909,26 @@ def check_connection(self, rhel_pod_obj, host, pem_dst_path):
             host, pem_dst_path, cmd, user=constants.EC2_USER
         )
 
+    def get_zone_number(self):
+        """
+        Get a zone number out of all available zones, we are looking for
+        sequential iteration over available zones. Note: This is not exact
+        zone info of aws, instead we are looking for an index to the
+        cloudformation stack whose zone info will be extracted later.
+
+        Returns:
+            int: Zone number, each call to this function returns next avaialable
+                zone number from available zones
+
+        """
+        num_zones = get_az_count()
+        try:
+            self.zone_count += 1
```

Comment:
as we discussed on our mtg, I would use class member here so it can be shared among all the instances and will give us better distribution of AZs for whatever call from whatever instance.

see example:

```
In [17]: class ClassMembersTest():
    ...:     my_class_member = 0
    ...:
    ...:     def increase_value(self):
    ...:         ClassMembersTest.my_class_member += 1
    ...:
    ...:
    ...:

In [18]: instance1 = ClassMembersTest()

In [19]: instance2 = ClassMembersTest()

In [20]: instance1.my_class_member
Out[20]: 0

In [21]: instance2.my_class_member
Out[21]: 0

In [22]: instance1.increase_value()

In [23]: instance2.my_class_member
Out[23]: 1

In [24]: instance1.my_class_member
Out[24]: 1

In [25]: instance2.increase_value()

In [26]: instance1.my_class_member
Out[26]: 2

In [27]: instance2.my_class_member
Out[27]: 2
```